pkg/operator/ceph/cluster/cluster.go

```diff
@@ -88,7 +88,7 @@ func newCluster(c *cephv1.CephCluster, context *clusterd.Context, csiMutex *sync
 	}
 }
 
-func (c *cluster) doOrchestration(rookImage string, cephVersion cephver.CephVersion, spec *cephv1.ClusterSpec) error {
+func (c *cluster) deploy(rookImage string, cephVersion cephver.CephVersion) error {
```

Comment:
Something more specific might be helpful, we have so many other methods about reconciling. How about this?
```suggestion
func (c *cluster) reconcileCephDaemons(rookImage string, cephVersion cephver.CephVersion) error {
```